content/file-system/slides/_index.md

```diff
@@ -212,7 +217,7 @@ You can try this out by creating a new file with `touch example-file` checking t
 
 Reading the contents of a file is always handy. In a terminal you can either take the entire contents and dump it into STDOUT with `cat`.
 
-Or you can load pieces of the file into RAM at a time with `less`.
+Or you can load the file into RAM pieces at a time with `less`.
```

Comment:
I added some clarity here. Using the term chunks is ideal because that is the common nomenclature when referring to memory (RAM). But I agree it needs to be cleaned up, which I did.